packages/core/src/app/guestSignup/GuestSignUpForm.test.tsx

```diff
@@ -0,0 +1,109 @@
+import userEvent from '@testing-library/user-event';
+import { noop } from 'lodash';
+import React from 'react';
+
+import { createLocaleContext, LocaleContext, LocaleContextType } from '@bigcommerce/checkout/locale';
+import { render, screen } from '@bigcommerce/checkout/test-utils';
+
+import { getStoreConfig } from '../config/config.mock';
+
+import GuestSignUpForm from './GuestSignUpForm';
+
+describe('GuestSignUpForm', () => {
+    const localeContext: LocaleContextType = createLocaleContext(getStoreConfig());
```

Comment:
We don't need `LocaleContextType` if we are assigning it straightaway.

Reply:
It's mainly for `const translate = localeContext.language.translate;`, otherwise, I think we may not be able to do `
        expect(screen.getByText(translate('customer.create_account_action'))).toBeInTheDocument();`.

Reply:
For example this https://github.com/bigcommerce/checkout-js/pull/2255/files#diff-ba354ac3db6c5387d43cc02cd7ac4f75059ad773260ae60fe6d58f71dc6e4d5bR14.

Reply:
I misunderstood...thought you were referring to removing `localeContext` here.
Will remove the type in the next PR.